.github/workflows/test.yml

```
@@ -161,3 +150,15 @@ jobs:
         path: "test_output/*.junit.xml"
         type: "junit"
         githubToken: ${{ secrets.GITHUB_TOKEN }}
+
+  test-windows:
+    runs-on: windows-latest
+    steps:
+    - uses: actions/checkout@v3
+    - uses: actions/setup-node@v3
+      with:
+        node-version: 14
```

lest make it faster 😄 
```suggestion
        node-version: 14
        cache: 'npm'
```